app/Category.php

```diff
@@ -6,5 +6,7 @@
 
 class Category extends Model
 {
-    //
+    public function products(){
+        return $this->belongsToMany('App\Product');
```

Comment:
return $this->belongsToMany(Product::class);

Reply:
https://laravel.com/docs/5.8/eloquent-relationships#one-to-many-inverse
Tại sao lại viết như trên thế anh.Em thấy trên tài liệu laravel người ta hướng dẫn viết như thế ''$this->belongsToMany('App\Product');"mà anh.